src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/MålgruppeValidering.kt

```diff
@@ -0,0 +1,24 @@
+package no.nav.tilleggsstonader.sak.vilkår.vilkårperiode
+
+import no.nav.tilleggsstonader.kontrakter.felles.Stønadstype
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.feilHvisIkke
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.MålgruppeType
+
+object MålgruppeValidering {
+
+    fun validerKanLeggeTilMålgruppeManuelt(stønadstype: Stønadstype, målgruppeType: MålgruppeType) {
```

Comment:
Burde vi ha dette for aktivitet også? Eller finnes det kanskje ingen ulike aktiviteter som vi vet om atm. Kan legges til senere

Reply:
Hva jeg vet er alle aktiviteter gyldige for alle stønadstyper